app/views/layouts/base.html.erb

```diff
@@ -149,33 +149,35 @@ See COPYRIGHT and LICENSE files for more details.
 </div>
 <%# Properly decides main menu expanded state and width before its drawn. Fixes flickering side menu
         where menu is first expanded, then being collapsed in angular. %>
-<%= nonced_javascript_tag do %>
+<%= nonced_javascript_tag defer: true do %>
```

Comment:
FYI setting the script's `defer` attribute - along with `window.addEventListener('turbo:load', () => {` is necessary to ensure jQuery has been loaded before this code is called.

Alternatively, we could investigate loading jQuery as a separate, external dependency (without the `type: "module"`):

    = javascript_include_tag "jquery", "data-turbo-track": "reload"